### DISCUSSION TITLE
10k download, Thanks All Brothers and Sisters

### DISCUSSION THREAD

**Discussion by tim-hub** [tim-hub/obsidian-bible-reference#113] [Announcements]
I have to admit last year, I didn't contribute to the project much, but thanks for your support, feedback and continue using it.

![image](https://github.com/tim-hub/obsidian-bible-reference/assets/2884320/2085a9bc-1e74-4087-87b5-2491dd549353)

Total download, it reaches 10k+ now.

- As a full time software engineer, sometimes it is just hard to manage time for community/open source contribution, last year for me, it is a lot of my life and career changes, but I would like to say thanks again, and I would like to come back to make more contribution here.

Thanks for all the contributors. Especially for @JPT62089, I know last year you took a lot of responsibility to not only making code changes, resolving issues, but also actively contribute in discussion.

Thanks for all the previous sponsors too, @PatrickMcGrath29 @chrisg32 @aknisly Thanks for the trust and support.

Anyway, it is late in my timezone, but all the best to you all, and let me fix the issues, and bring more versions and more features to the plugin. 

Thanks all.

> Matthew 6:34 ESV
> “Therefore do not be anxious about tomorrow, for tomorrow will be anxious for itself. Sufficient for the day is its own trouble.

**Comment by chrisg32** [comment]
Thank you @tim-hub for the work you have put into this extension. For me it is an invaluable tool in my daily Bible and systematic theology studies. Please keep being awesome!

**Reply by tim-hub** [reply]
Hi @chrisg32 

I am building a web page for the plugin, I would like to quote your comment to use as testimony in the plugin description. 

Please let me know if it is OK to do so. :)

**Comment by kyle-mirich** [comment]
Hey @tim-hub  thank you Brother I just got the project up and running in my obsidian notes and it works amazing! This is exactly what I was looking for  to take my Bible study notes. God Bless!

**Reply by tim-hub** [reply]
Thanks

**Reply by tim-hub** [reply]
hi @kyle-mirich 


I am building a web page for the plugin, I would like to quote your comment to use as testimony in the plugin description. 

Please let me know if it is OK to do so. :)

**Comment by SamejSpenser** [comment]
Hey @tim-hub, I just wanted to thank you for your invaluable work on the OBR plugin, it’s been indispensable in my daily Bible studies for several months now.

May God bless you powerfully.
Grace and Peace! 🙏

**Reply by tim-hub** [reply]
Thanks @SamejSpenser 

Is it OK for me to quote your comments to put on the website?

**Reply by SamejSpenser** [reply]
Hello again @tim-hub; yes, you are completely free to quote my comments on the website! 😉

**Reply by chrisg32** [reply]
Absolutely you are welcome to quote me.

Thank you

**Comment by kyle-mirich** [comment]
Your more than welcome. Thanks Tim 🙏🏻
